### PR TITLE
suppress/revive

### PR DESCRIPTION
**Pull request by mgummelt**

### CODE REVIEW

File: integration/tests/test_sanity.py [hunk 1]
```diff
@@ -78,3 +78,10 @@ def test_connect_dns(install_framework):
     except:
         print('Failed to parse connect response')
         raise
+
+
+@pytest.mark.sanity
+def test_is_suppressed():
+    response = dcos.http.get(cassandr_api_url('service/kafka/v1/state/properties/suppressed'))
```
Comment (verma7) [inline integration/tests/test_sanity.py]: Should this be 'service/cassandra/...'?

Comment (mgummelt) [inline integration/tests/test_sanity.py]: yep!  thanks.

**Comment by mgummelt** [comment]
retest this please

**Comment by mgummelt** [comment]
retest this please

**Comment by mgummelt** [comment]
retest this please

**Comment by mgummelt** [comment]
retest this please

**Comment by mgummelt** [comment]
cc @mohitsoni tests pass.  plz review

File: integration/tests/command.py [hunk 2]
```diff
@@ -122,3 +122,28 @@ def uninstall():
             )[0].strip()
         )
     )
+
+
+# def get_and_verify_plan(predicate=lambda r: True):
```
Comment (mohitsoni) [inline integration/tests/command.py]: Can we remove this unused code ?

**Comment by mohitsoni** [comment]
LGTM! We can merge after we remove the unused code from `command.py`

**Pull request merged**